### PR TITLE
Web troubleshooting

### PR DESCRIPTION
**Pull request by sbesson**
This PR:

- remove out of date OMERO.web troubleshooting sections
- adds a section about the `Permission denied` error with the default Apache stanza

/cc @manics @aleksandra-tarkowska

### CODE REVIEW

**Comment by manics** [comment]
Could we change the perms on `var/` instead?

File: omero/sysadmins/troubleshooting.txt [hunk 1]
```diff
@@ -237,55 +237,18 @@ wrong. See :ref:`search-failures` for more details.
...
-To configure the out-of-the box setup to listen for webadmin and webclient
-connections on different host run:
-
-::
-
-    c:\omero_dist> bin/omero web start 'host' 'port'
-
-
 OMERO.web did not start on the production
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (rleigh-dundee) [inline omero/sysadmins/troubleshooting.txt]: "one the production"... what?  This is incomplete.

**Comment by rleigh-dundee** [comment]
The text reads OK to me, links are correct etc.  Just the minor nit about the subheading being incomplete, otherwise looks good to merge.

**Comment by hflynn** [comment]
Malformed heading isn't @sbesson's fault and I'm going to open a new PR editing this page anyway so I can fix it there.

**Comment by hflynn** [comment]
@sbesson ok for me to merge this or did you want to address @manics' point?

**Comment by sbesson** [comment]
@manics's point should be addressed in a top-level (openmicroscopy) PR. I will try to open one for discussion today.

**Comment by hflynn** [comment]
Great, I'll merge this for now then, this page can always be updated again post the relevant code PR.

**Pull request merged**